src/coverlet.core/Symbols/CecilSymbolHelper.cs

```diff
@@ -279,6 +362,13 @@ Single branch
             return true;
         }
 
+        internal static Instruction GetPreviousNoNopInstruction(Instruction current)
+        {
+            Instruction instruction = current.Previous;
+            for (; instruction != null && instruction.OpCode == OpCodes.Nop; instruction = instruction.Previous) { }
```

Comment:
I don't like empty body can you invert loop condition and return from inside the body?